connector/elasticapmconnector/README.md

```diff
@@ -39,11 +39,43 @@ By default, aggregated metrics will be exported without any client metadata. It
 propagate client metadata from input to exported metrics by specifying a list of metadata keys
 in `elasticapm::aggregation::metadata_keys`.
 
+By default, cardinality for aggregated metrics will be limited.
+Each limit defines a `max_cardinality` and a list of `attributes` that will be added to the overflow metric.
+There are four limits that can be configured: 
+- `elasticapm::aggregation::resource_limit`: configures the max cardinality of resources
+- `elasticapm::aggregation::scope_limit`: configures the max cardinality of scopes within a resource
+- `elasticapm::aggregation::metric_limit`: configures the max cardinality of metrics within a scope
+- `elasticapm::aggregation::datapoint_limit`: configures the max cardinality of datapoints within a metric
+
 ```yaml
 elasticapm:
   aggregation:
     directory: /path/to/aggregation/directory
     metadata_keys: [list, of, metadata, keys]
+    resource_limit:
+      max_cardinality: 8000
+      overflow:
+        attributes:
+          - key: "overflow"
+            value: true
+    scope_limit:
+      max_cardinality: 4000
+      overflow:
+        attributes:
+          - key: "overflow"
+            value: true
+    metric_limit:
+      max_cardinality: 4000
+      overflow:
+        attributes:
+          - key: "overflow"
+            value: true
```

Comment:
I don't think this is right, could you double check the attributes that are published for apm-aggregation for overflow bucket: https://github.com/elastic/apm-aggregation/blob/main/aggregators/converter.go

We should keep them same as the UI relies on it. https://github.com/elastic/opentelemetry-collector-components/issues/141#issuecomment-2592951016 comment can probably aid in understanding how lsminterval overflow can be used for achieving parity with apm-aggregation

Reply:
Ah okay I did not know the UI relies on certain overflow attribute values. I will add `_other` in the appropriate places

Reply:
I updated to define only the `service.name: _other` attribute. The apm-aggregator defines separate overflow attributes for `transaction.name` and `transaction.type`. We would have to always apply these attributes for every metric which I do not think we want. For example the `service_summary` metric should probably not have these transaction attributes. I am double checking with the kibana team if they still only rely on  `service.name: _other`

Reply:
@lahsivjar  I updated the connector only have one default overflow attribute for now `service.name: _other`. 

I created https://github.com/elastic/opentelemetry-collector-components/issues/697 since I could not find a simple way to only apply `transaction.name` for span aggregation metrics.

Reply:
Sounds good, however take a look at the comment I posted on where the `service.name` should be overridden.

For adding overflow attriburtes to other entities, we had plans to do this via OTTL conditions but that was not done yet. For now, we can add a generic identifier to those so that we can debug overflows and in future (once #697 is addressed) we can update those configs.